qutip/core/cy/qobjevo.pyx

```diff
@@ -482,7 +482,16 @@ cdef class QobjEvo:
                     f"Desired feedback {key} is not available for the {solver}."
                 )
             new_args[key] = solvers_feeds[feed]
-        self.arguments(**new_args)
+        # self.arguments(new_args)
```

Comment:
How did calling `.arguments(...)` wipe out the feedback items? What is different about calling `.replace_arguments`?

I guess this commented out line should now be deleted.

Reply:
It is set so that `argument` can change from a normal args to a feedback and vise-versa. 
```
qevo = QobjEvo(..., args={"A": MESolver.ExpectFeedback(op))
solver = MESolver(qevo, c_ops)
solver.run(...) # Run with feedback
solver.run(..., args={"A": 1.) # Rerun using the same QobjEvo without feedback
solver.run(..., args={"A": MESolver.StateFeedback(rho0)) # Rerun using new feedback
```
Therefore when `argument` is called, it update the `_feedback_functions` and `_solver_only_feedback` attributes to reflect the change between normal and dynamic args.

`replace_arguments` apply the new values of the args on the `coefficients`, but do not change attributes of the `QobjEvo`.

Reply:
```suggestion
```

Reply:
Thanks, that makes sense.